addons/test_mail/tests/test_mail_followers.py

```diff
@@ -472,6 +472,28 @@ def test_auto_subscribe_defaults(self):
             'AutoSubscribe: at create auto subscribe as creator + from parent take both subtypes'
         )
 
+    def test_subtype_parent_same_model(self):
+        subtype_1 = self.env['mail.message.subtype'].create({
+            'name': '',
+            'res_model': 'res.partner',
```

Comment:
I know we can be annoying, but could you use test models ? We have quite a lot of them, could be worth a check :) we notably have mail.test.ticket model, that somehow looks like tasks / helpdesk tickets. Or just use some track model that exist (look at ``mail.test.track.*`` models).